### PR TITLE
Remove strang nse

### PR DESCRIPTION
**Pull request by zingale**
Strang does not really allow us to do 2nd order easily because we can't capture the evolution of the state during the burn

### CODE REVIEW

**Comment by zingale** [comment]
addresses #1542

**Comment by zingale** [comment]
I think that this is good now

File: integration/Make.package [hunk 1]
```diff
@@ -26,11 +26,12 @@ else
   CEXE_headers += integrator_setup_strang.H
 endif
 
+
 ifeq ($(USE_NSE_TABLE), TRUE)
```
Comment (zhichen3) [inline integration/Make.package:30]: should we include USE_NSE_NET here?

Comment (zingale) [reply integration/Make.package]: fixed

**Review by zhichen3 — APPROVED** [review]

**Pull request merged**